Source/Search/Advanced.lua

```diff
@@ -0,0 +1,120 @@
+function Auctionator.Search.IsAdvancedSearch(searchString)
+  return Auctionator.Utilities.StringContains(searchString, Auctionator.Constants.AdvancedSearchDivider);
+end
+
+-- Extract components of an advanced search string. Assumes searchString is an
+-- advanced search.
+function Auctionator.Search.SplitAdvancedSearch(searchString)
```

Comment:
I set this up so that the code using the advanced search wouldn't be affected by parameter order in the search term, and couldn't accidentally confuse level and ilvl

Reply:
Makes sense.